src/main/resources/alfresco/templates/webscripts/org/alfresco/repository/upload/upload.post.config.xml

```diff
@@ -1,5 +1,4 @@
 <upload>
-<!-- modify this configuration to change the versioning behaviour for this API - remove the section to apply content model default behaviour -->
-  <autoVersion>true</autoVersion>

```

Comment:
Perhaps leave this config in place but commented out (after your new comment), so that there is an example of how to apply it back; I do not think anyone will since the behaviour is adjustable from the config but your new comment indicates that it can be done but does not show how.